third_party/move/move-compiler-v2/src/pipeline/ability_checker.rs

```diff
@@ -272,7 +291,7 @@ fn check_bytecode(target: &FunctionTarget, code_offset: CodeOffset, bytecode: &B
                 BorrowField(mod_id, struct_id, insts, _) => {
                     check_struct_inst(target, *mod_id, *struct_id, insts, &loc);
                 },
-                Drop => cond_check_drop_for_temp_with_msg(
+                Drop | Release => cond_check_drop_for_temp_with_msg(
```

Comment:
The semantics of `Release` is that a value is no longer borrowed, but it could still be used later?